.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+{
+  // hide cache files
```

Comment:
The settings look pretty clear to me so maybe the comments are not necessary.

Reply:
I can remove them. It does always feel weird using json5 in a `.json` file anyway

Reply:
oh.. [JSON5](https://json5.org/), new to me. 🤓 
maybe only the 
```
// isort
```
comment is necessary since it's not mentioned in the settings.